### PR TITLE
Less, and less-revealing, logging in staging

### PR DESCRIPTION
**Pull request by rogerz42892**

### CODE REVIEW

File: lib/rate_limiting.rb [hunk 1]
```diff
@@ -94,9 +94,9 @@ def logger
   end
 
   def allowed?(request)
-    logger.debug "Rate Limit REQ: #{request.host} #{request.path} #{request.params} #{request.url}"
+    logger.debug "Rate Limit REQ: #{request.host} #{request.path} #{request.params} #{request.url}" if Rails.env.development?
```
Comment (rogerz42892) [inline lib/rate_limiting.rb:97]: request.params is the worst-offender - won't show up at all in staging.

File: lib/rate_limiting.rb [hunk 2]
```diff
@@ -94,9 +94,9 @@ def logger
...
     if rule = find_matching_rule(request)
-      logger.info "Rate Limit MATCH: #{request.url}"
+      logger.info "Rate Limit MATCH: #{request.path}" if Rails.env.development?
```
Comment (rogerz42892) [inline lib/rate_limiting.rb:99]: This is really just for validating the rules, so dev is fine.

File: lib/rate_limiting.rb [hunk 3]
```diff
@@ -119,7 +119,7 @@ def apply_rule(request, rule)
     key = rule.get_key(request)
     if cache_has?(key)
       record = cache_get(key)
-      logger.info "Rate Limit ENTRY #{request.url}: '#{key}' => #{record}"
+      logger.info "Rate Limit ENTRY #{request.path}: '#{key}' => #{record}"
```
Comment (rogerz42892) [inline lib/rate_limiting.rb:122]: These are nice to have in production (and staging), but they won't show any params at all now.

**Review by ozydingo — APPROVED** [review]
:shipit:

**Pull request merged**